extra/1-radio-stations.js

```diff
@@ -15,6 +15,15 @@
 
 // `getAllFrequencies` goes here
 
+function getAllFrequencies(){
+  let array = [];
+    for (i = 87; i < 109; i++) {
```

Comment:
Hi! I am reviewing your code, and it looks like you know a lot more than me, so I don't think I can add anything, but i sure learned a few things going through your code so thank you! Nice use of for loop here.